data/DevOps_Examples/2189.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I love the sample data but I'm worried about muddying up the sample index with unrelated data. I suppose we already do that for the GPT4V examples, and it hasn't practically caused an issue, so maybe that's fine? @mattgotteiner Thoughts?

Might be helpful to think about whether we're going to end up adding example HTML, docx, pptx, etc. Probably not as it slows ingestion time and overall deployment time. That's another reason to not add it.

If we do keep it, I'd suggest naming the folder to JSON_Examples since that's more their purpose in the repository.

Reply:
I just tested this. Parsing those JSON files takes less than 30 seconds, so maybe we just keep multiple example files. Its the PDFs and the call to Doc Intel that adds significantly to the time.